DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -31,5 +33,43 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
                 new { customisationId }
             );
         }
+
+        public int GetProgressId(int candidateId, int customisationId)
+        {
+            return connection.QueryFirstOrDefault<int>(
+                @"SELECT ProgressId
+                        FROM Progress
+                        WHERE CandidateId = @candidateId
+                          AND customisationId = @customisationId",
+                new { candidateId, customisationId }
+            );
+        }
+
+        public void UpdateLoginCountAndDuration(int progressId)
+        {
+            var numberOfAffectedRows = connection.Execute(
+                @"UPDATE Progress
+	                    SET LoginCount = (SELECT COALESCE(COUNT(*), 0)
+		                    FROM Sessions as s

```

Comment:
Another couple of consistency things:
* We generally write any keywords (such as "AS") in uppercase 
* We generally capitalise the table aliases, so "S" instead of "S"

🙂